packages/mux-audio/package.json

```diff
@@ -44,6 +44,7 @@
     "build": "npm-run-all --parallel build:types 'build:esm -- --minify' 'build:iife -- --minify' 'build:cjs -- --minify' 'build:esm-module -- --minify'",
     "prebuild": "yarn clean",
     "prepublishOnly": "yarn build",
+    "deploy:canary": "npm version $(semver $(npm view @mux-elements/mux-audio versions --json | jq -r '. - map(select(contains(\"beta\"))) | last' | sed -r 's/-[a-z0-9]{7}$//g') -i prerelease --preid canary)-$(git rev-parse --short HEAD) --git-tag-version false && npm publish --tag canary",
```

Comment:
so, the jq command filters all the versions that contain "beta" version and then removes them from the list. Should we add `alpha` in here too?

Reply:
yes, it shouldn't semver increase based on other prereleases. 
yes, if we would use `alpha` it should be added, will add to be safe